index.html

```diff
@@ -0,0 +1,237 @@
+                <h1 class="works-primary-text"
+                >Tonic
+                </h1>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="button">See Project</button>
+            </h2>
+        </h2>
+        <h2 class="card-container app-bckground-color">
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio 4.png" alt="Card Image">
+                <h1 class="works-primary-text"
+                >Multi-Post <br/> Stories 
+                </h1>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul> 
+                <button type="button">See Project</button>
+            </h2>
+        </h2>
+    </h2>
+</section>
+<section class="about app-bckground-color" id="about">
+    <h2 class="about-myself app-margin">
+        <h1 class="about-primary-text"
+        > About <br> myself 
+        </h1>      
+        <p class="about-primarytext">
+            Hello I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <h3 class="">
+            LET’S CONNECT
+        </h3>
+        <ul class="about-myself-social-media-icons">
+            <li><a href="#"><img src="twitter-icon.png" alt="Twitter"></a></li>
+            <li><a href="#"><img src="Linkedin-icon.png" alt="LinkedIn"></a></li>
+            <li><a href="#"><img src="mail-icon.png" alt="Email"></a></li>
+            <li><a href="#"><img src="git-icon.png" alt="GitHub"></a></li>
+            <li><a href="#"><img src="chill-icon.png" alt="Chill"></a></li>
+        </ul>
+        <button type="button">Get My Resume</button>
```

Comment:
- [ ] Please ensure that you style your `button` according to the given design requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=hnNtRADBdqYtrcFO-0) and `Implement the button interactions (enable, hover, etc.)`. :slightly_smiling_face: 

Below is the screenshot showing the difference between yours and the Figma design. :arrow_down: 


<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student Margin](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/8c476262-c26e-4c22-8e43-eb4dffda1e1f)


</td>
      <td>

![Design Margins](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/ad72620c-41e1-4a45-a30d-2b6a14d16a5d)

</td>
    </tr>
  </table>

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student Card](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/e43294a7-b418-4dc4-a475-6008c800d9d9)

</td>
      <td>

![design card](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/37f42b4b-5ed8-42c6-b77e-20c53bba356c)

</td>
    </tr>
  </table>

<table>
    <tr>
      <td>Your design (applies to all your Cards)</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student Card](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/a5b033e1-8aac-4e72-94c5-9f5f8bf68eda)

</td>
      <td>

![design card](https://github.com/wachirasenior0/portfolio-setup-and-mobile-first/assets/43528594/6ac8f575-ef03-4bd4-8943-debf36af0598)
</td>
    </tr>
  </table>